conda_build_config.yaml

```diff
@@ -1,2 +1,2 @@
-c_stdlib_version:  # [osx]
```

Comment:
Please don't touch this.

Reply:
This file was changed 8h ago and i believe the current formatting is broken